README.md

```diff
@@ -1,28 +1,43 @@
-# purescript-ace
+# Ace
 
-[![Latest release](http://img.shields.io/github/release/slamdata/purescript-ace.svg)](https://github.com/slamdata/purescript-ace/releases)
-[![Build status](https://travis-ci.org/slamdata/purescript-ace.svg?branch=master)](https://travis-ci.org/slamdata/purescript-ace)
-[![Dependency status](https://img.shields.io/librariesio/github/slamdata/purescript-ace.svg)](https://libraries.io/github/slamdata/purescript-ace)
+[![CI](https://github.com/purescript-contrib/purescript-ace/workflows/CI/badge.svg?branch=main)](https://github.com/purescript-contrib/purescript-ace/actions?query=workflow%3ACI+branch%3Amain)
+[![Release](http://img.shields.io/github/release/purescript-contrib/purescript-ace.svg)](https://github.com/purescript-contrib/purescript-ace/releases)
+[![Pursuit](http://pursuit.purescript.org/packages/purescript-ace/badge)](http://pursuit.purescript.org/packages/purescript-ace)
+[![Maintainer: milesfrain](https://img.shields.io/badge/maintainer-milesfrain-teal.svg)](http://github.com/milesfrain)
```

Comment:
Who wants to be the maintainer?

Reply:
If there is no maintainer assigned, then by default you can assign it to Gary and myself as we're responsible for these libraries in general. However, later this year we'd like to do a more formal process to identify other maintainers for these libraries.